### PR TITLE
Adding czech language translation.

### PR DESCRIPTION
**Pull request by jbenak8**

### CODE REVIEW

**Comment by palexdev** [comment]
Hello @jbenak8, thanks for your contribution
I think you have missed some values though:
```
comboBox.contextMenu.selectNext=Select next
comboBox.contextMenu.selectPrevious=Select previous
comboBox.contextMenu.selectLast=Select last
comboBox.contextMenu.clearSelection=Clear selection
```

File: materialfx/src/main/resources/io/github/palexdev/materialfx/i18n/mfxlang_cs_CZ.properties [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+##################################################
+# Controls
+##################################################
+
+# Combo Box
+comboBox.contextMenu.selectFirst=Vybrat první
+comboBox.contextMenu.selectNext=Select next
+comboBox.contextMenu.selectPrevious=Select previous
+comboBox.contextMenu.selectLast=Select last
+comboBox.contextMenu.clearSelection=Clear selection
```
Comment (GrolarDan) [inline materialfx/src/main/resources/io/github/palexdev/materialfx/i18n/mfxlang_cs_CZ.properties:29]:
```suggestion
comboBox.contextMenu.selectNext=Vybrat další
comboBox.contextMenu.selectPrevious=Vybrat předchozí
comboBox.contextMenu.selectLast=Vybrat poslední
comboBox.contextMenu.clearSelection=Zrušit výběr
```

**Comment by jbenak8** [comment]
Hi @palexdev thank you for noticing me missing traslations.
Hix @GrolarDan many thanks for checking it for typos.

**Review by GrolarDan — APPROVED** [review]
👍

**Comment by palexdev** [comment]
Integrated, thanks!

**Pull request merged**